gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -1508,7 +1508,8 @@ void
 Dump::visit (TypeParam &e)
 {
   begin ("TypeParam");
-  put_field ("outer_attr", e.get_outer_attribute ().as_string ());
+  auto oa = e.get_outer_attrs ();
+  do_outer_attrs (oa);
```

Comment:
```suggestion
  auto &outer_attrs = e.get_outer_attrs ();
  do_outer_attrs (outer_attrs);
```

Reply:
I think the extra `&` is necessary to avoid making a copy, not that it really matters here

Reply:
Good catch, I think this was a holdover from before we were copying vectors. I fixed this